### DISCUSSION TITLE
A doubt I have always had...🤔

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/stable-diffusion-webui-forge#1329] [Q&A] [answered]
When we are generating an image and we change something in the prompt, we need to interrupt the generation and start again, or it takes the new words in the prompt on the "fly"?

**Comment by Liva24** [comment, marked as answer]
Yes you have to interrupt and start it again. Sometimes while it is generating a few images im changing the promts I wanna try next, no waist of time haha
